easybuild/easyconfigs/k/kokkos/kokkos-3.4.01-GCC-10.3.0.eb

```diff
@@ -0,0 +1,43 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'kokkos'
+version = '3.4.01'
+
+homepage = 'https://kokkos.org'
+description = """Kokkos Core implements a programming model in C++ for writing
+                 performance portable applications targeting all major HPC
+                 platforms. For that purpose it provides abstractions for both
+                 parallel execution of code and data management. Kokkos is
+                 designed to target complex node architectures with N-level
+                 memory hierarchies and multiple types of execution resources.
+                 It currently can use CUDA, HPX, OpenMP and Pthreads as backend
+                 programming models with several other backends in development."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/%(name)s/%(name)s/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['146d5e233228e75ef59ca497e8f5872d9b272cb93e8e9cdfe05ad34a23f483d1']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+]
+
+dependencies = [
+    ('binutils', '2.36.1'),
```

Comment:
No need for the binutils dependency when using `GCC` as toolchain?

Reply:
Ok, resolved!